source/s3_meta_request.c

```diff
@@ -242,26 +449,28 @@ PyObject *aws_py_s3_client_make_meta_request(PyObject *self, PyObject *args) {
     PyObject *s3_client_py = NULL;
     PyObject *http_request_py = NULL;
     int type;
-    PyObject *credential_provider_py;
+    PyObject *credential_provider_py = NULL;
+    const char *recv_filepath;
+    Py_ssize_t recv_filepath_len;
+    const char *send_filepath;
+    Py_ssize_t send_filepath_len;
```

Comment:
trivial: since we don't actually need the len, you can get rid of `recv_filepath_len` and `send_filepath_len`. Use `z` instead of `z#` down in PyArg_ParseTuple()